### PR TITLE
Plans 2023 : Removed feature group innovative features and other updates

### PR DESCRIPTION
**Pull request by ddc22**
#### Proposed Changes

- Removed group innovative technologies, and reorganised features and updated changes done to the list
- Updates to group titles as shown in the design file
image

#### Testing Instructions
- Go to /start/plans?flags=onboarding/2023-pricing-grid
- The groups and features should be displayed correctly in the correct order and grouping

#### Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

Fixes Automattic/martech#1433

### CODE REVIEW

**Comment by ddc22** [comment]
@claudiucelfilip can you please have another look at this PR?

File: packages/calypso-products/src/feature-group-plan-map.ts [hunk 1]
```diff
@@ -70,20 +69,22 @@ import { FeatureGroupMap } from './types';
 export const featureGroups: Partial< FeatureGroupMap > = {
 	[ FEATURE_GROUP_GENERAL_FEATURES ]: {
 		slug: FEATURE_GROUP_GENERAL_FEATURES,
```
Comment (aneeshd16) [inline packages/calypso-products/src/feature-group-plan-map.ts]: Nit: What do you think about renaming the slug to `FEATURE_GROUP_ESSENTIAL_FEATURES`? Might reduce confusion in the future. 🤔

**Review by aneeshd16 — APPROVED** [review]
Overall looks good, one difference that I found between the design and implementation:

**Design**
image

**Implementation**
image

Please feel free to ignore if this has been discussed already!

File: packages/calypso-products/src/feature-group-plan-map.ts (hunk 1 above)
Comment (ddc22) [reply packages/calypso-products/src/feature-group-plan-map.ts]: Yes good call I will do this.

**Review by claudiucelfilip — APPROVED** [review]
Features now match. Thank you!

LGTM

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7784520

Thank you @jdc91 for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7784520).

**Pull request merged**